docs/index.html

```diff
@@ -68,22 +68,22 @@ <h2 id="getting-started">Getting started</h2>
 <h3 id="installation-with-zef">Installation (with zef)</h3>
 <p>If you're just planning to be an Alma end user, <code>zef</code> is the recommended way to install Alma:</p>
 <div class="sourceCode"><pre class="sourceCode sh"><code class="sourceCode bash"><span class="ex">zef</span> install alma</code></pre></div>
-<p>In order to get the <code>zef</code> installer, you first need <a href="https://perl6.org/downloads/">Rakudo Perl 6</a>. Instructions for how to install <code>zef</code> itself can be found in the <a href="https://github.com/ugexe/zef#installation"><code>zef</code> README</a>.</p>
+<p>In order to get the <code>zef</code> installer, you first need <a href="https://raku.org/downloads/">Rakudo Raku</a>. Instructions for how to install <code>zef</code> itself can be found in the <a href="https://github.com/ugexe/zef#installation"><code>zef</code> README</a>.</p>
```

Comment:
I think the link text should just say "Rakudo", not "Rakudo Raku".

Reply:
Sure masak, I've updated this.